io.catenax.individual_asset_definition/1.0.0/IndividualAssetDefinition.ttl

```diff
@@ -0,0 +1,79 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.individual_asset_definition:1.0.0#>.
+@prefix ext-dimension: <urn:bamm:io.catenax.shared.physical_dimension:1.0.0#>.
+
+:IndividualAssetDefinition a bamm:Aspect;
+    bamm:preferredName "Individual Asset Definition"@en;
+    bamm:description "Permanent characteristics of an individual asset."@en;
+    bamm:properties (:catenaXId :assetOwnerId :assetSerialNumber :assetType ext-dimension:physicalDimensionsProperty);
```

Comment:
does it validate, if this is "just" written in the linked properies? @bs-jokri

Reply:
@catroest yes, this is syntactically valid. However, the question is if the semantics are ok, ie the description of the property fits the use at hand. @Cvetanka-ZF wdyt. does it fit?

Reply:
Yes, it does fit our use case. As for the validation, it did validate in the Aspect editor without problems. However an error is thrown when trying to generate an example JSON Payload. My colleague Brgit (@birgitZF) told me that this is a known issue in the editor.